src/liballoc/boxed.rs

```diff
@@ -6,6 +6,68 @@
 //!
 //! # Examples
 //!
+//! A pointer type for heap allocation.
```

Comment:
It looks like our docs might still be a little confused here, because they've changed on `master` since you fixed them up.

Would you like to try squashing your 12 commits down and rebase on `master`?